gnosis/safe-react
# Fetch apps from external resource

For the first iteration we are going to use a static file in order to move the app list. This static file is generated in this repository:
https://github.com/gnosis/safe-apps-list

In `/public` folder is the generated apps file
In `/config` the apps can be changed or updated with just the same parameters we used in the frontend list

When executing `yarn build` a new list is generated using the new configuration parameters. It automatically fetch the information exposed by the `manifest.json`  provided by each app

#### How to test
 - We should be able to load the apps list normally
 - We should be able to add a custom app normally
 - We should have the expected list (with the manually added app) after a full reload or closing and opening the page again
 - We should be able to open any app and execute a transaction from it
 - We should be able to do the same from the manually added app
 - We should be able to delete the manually added app
 - We shouldn't see the delete button in default apps
